dev/breeze/src/airflow_breeze/utils/shared_options.py

```diff
@@ -0,0 +1,61 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import annotations
+
+import os
+
+__verbose_value: bool = os.environ.get("VERBOSE", "false")[0].lower() == "t"
+
+
+def set_verbose(verbose: bool):
+    global __verbose_value
+    __verbose_value = verbose
+
+
+def get_verbose(verbose_override: bool | None = None) -> bool:
+    if verbose_override is None:
+        return __verbose_value
+    return verbose_override
```

Comment:
Not really. VerboseOverride has 3 values - True/False/None. It might also be False (in which case it woudl not work). Hence we need expicit "is None" check (which will not pass if vernose_override is False.